docs/tutorials/qpe_with_sampler.ipynb

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
<p>Can we include a line after the subheader about what the intention of the next box is? Or do you think the paragraph above covers the circuit creation?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/404/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Yes I will add at least a line of text after each subheader.</p>